apps/web/src/components/invoice-generator/index.tsx

```diff
@@ -40,13 +40,16 @@ function InputRow({
   autoComplete,
   type,
   defaultValue,
+  value,
   multiple,
   required,
   step,
   min,
-  onBeforeInput: onInput,
+  onBeforeInput,
+  onInput,
   unit,
   maxLength,
+  readOnly,
```

Comment:
Should we pass all these props directly to the input instead of mapping it 1-by-1 when nothing is changed about the props?

Like

```ts
function InputRow({
  id,
  ...restProps
}: GenericFieldProps) {
// ...

<Input
  id={htmlId}
  type={type ?? "text"}
  {...restProps}
  >
```